db/schema.rb

```diff
@@ -56,12 +55,12 @@
     t.inet     "last_sign_in_ip"
     t.datetime "created_at",                             null: false
     t.datetime "updated_at",                             null: false
-    t.integer  "points"
     t.string   "first_name"
     t.string   "last_name"
     t.string   "confirmation_token"
     t.datetime "confirmed_at"
     t.datetime "confirmation_sent_at"
+    t.integer  "points",                 default: 0
```

Comment:
This seems like a change I made and should already be in `master`.  It's not destructive though so I don't think we need to spend any time debugging it.